src/cassandra.go

```diff
@@ -74,6 +72,21 @@ func main() {
 	fatalIfErr(i.Publish())
 }
 
+func metricSet(e *integration.Entity, eventType string) *metric.Set {
```

Comment:
[super-minor] I prefer to pass arguments `host` and `port` instead of using global args, but it's just my preference.

Reply:
It's fixed, you're right it's better that way.